components/locale/en_US.ts

```diff
@@ -9,6 +9,9 @@ const typeTemplate = '${label} is not a valid ${type}';
 
 const localeValues: Locale = {
   locale: 'en',
+  a11y: {
+    sortableColumnHeaderAriaDescription: 'sortable column',
```

Comment:
`sortable` and make a common desc that mean support sort. Which we can use same locale in other compnent

Reply:
@zombieJ `sortable column` can only be used in Table.

Reply:
> @zombieJ sortable column can only be used in Table.

```tsx
// EN
ally: {
  sortable: 'Sortable',
}

// CN
ally: {
  sortable: '可排序',
}
```